### DISCUSSION TITLE
Is there a way to use sqlite remotely?

### DISCUSSION THREAD

**Discussion by johhansantana** [keystonejs/keystone#7950] [Questions]
For example, hosting the sqlite file in aws s3 and replace using the local one?

**Comment by VinylVault** [comment]
technically I would say this is possible, however definitely unadvisable, for latency reasons, also if the connection "Burps" for any reason the data could be corrupted, if the DB write is not checked for accuracy immediately after this could easily be a huge problem, if the S3 bucket is on your LAN using something like MinIO then you should be OK
